components/internsDAO/proposals.tsx

```diff
@@ -1,12 +1,85 @@
 // @ts-nocheck
-
+import snapshot from "@snapshot-labs/snapshot.js";
+import { useState, useEffect } from "react";
 
 function Proposals() {
-    return (
-        <div>
-            proposals
-        </div >
-    )
+  const [proposals, setProposals] = useState([]);
+  const space = "internsdao.eth";
+
+  ///-----Taking from https://docs.snapshot.org/snapshot.js-----////
+
+  // const hub = "https://testnet.snapshot.org";
+  // const client = new snapshot.Client712(hub);
+  // const vote = async (option, proposal) => {
+  //   const { account } = context;
+  //   const receipt = await client.vote(context, account, {
+  //     space: space,
+  //     proposal: proposal,
+  //     type: "single-choice",
+  //     choice: option,
+  //   });
+  // };
+
+  useEffect(() => {
+    fetch("https://testnet.snapshot.org/graphql", {
+      method: "POST",
+      headers: { "content-type": "application/json" },
+      body: JSON.stringify({
+        query: `query Proposals {
+        proposals(
+          first: 20,
+          skip: 0,
+          where: {
+            space_in: ["${space}"],
+            state: "active"
+          },
+          orderBy: "created",
+          orderDirection: desc
+        ) {
+          id
+          title
+          body
+          choices
+          start
+          end
+          snapshot
+          state
+          author
+          space {
+            id
+            name
+          }
+        }
+      }`,
+      }),
+    })
+      .then((res) => res.json())
+      .then((data) => setProposals(data.data.proposals))
+      .catch((err) => console.log(err));
+    // console.log(proposals);
+  }, []);
+
+  return (
+    <div>
+      {proposals &&
+        proposals.map((proposal) => (
+          <div style={proposalsStyle}>
```

Comment:
need to add key prop
![Screenshot_20221110_104201](https://user-images.githubusercontent.com/47399791/201029910-cddba3d9-6a94-4165-874c-986edd572429.png)